nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -45,6 +45,7 @@ enterprise/websvc.restlib/external/javax.annotation-api-1.2.jar java/maven.index
 enterprise/javaee.api/external/javax.annotation-api-1.2.jar java/maven.indexer/external/javax.annotation-api-1.2.jar
 enterprise/javaee7.api/external/javax.annotation-api-1.2.jar enterprise/javaee8.api/external/javax.annotation-api-1.2.jar
 enterprise/javaee8.api/external/javax.annotation-api-1.2.jar enterprise/websvc.restlib/external/javax.annotation-api-1.2.jar
+enterprise/jakartaee8.api/external/javax.annotation-api-1.3.5.jar
```

Comment:
This needs fixing before we can merge. It's causing the verify libs and licenses test to fail. Each line must contain two paths.  Is it meant to be a duplicate of something?  If so, needs to have the other path.  If not, then the whole line needs removing.

Check with `ant verify-libs-and-licenses`

Reply:
Thanks I will take a look now and I will be in touch.  I appreciate it.

Reply:
This line needs to be removed and then it repairs the verify-libs-and-licenses.  Do you want me to merge the remote branch into my local and make the change?  I don't want to add more commits if not needed.

Reply:
I can do it here.  Got it open at the moment.  Just wanted to check why that was added.  Thanks!